docs/developers-clear-containers-install.md

```diff
@@ -85,26 +91,39 @@ to allow their creation.
 
 ```bash
 $ sudo yum install -y glibc-static
-$ cd $GOPATH/src/github.com/clearcontainers/runtime/.ci/
+$ cd $GOPATH/src/github.com/clearcontainers/tests/.ci/
 $ ./install_virtcontainers.sh
 ```
 
-## Enable Clear Containers 3.0 for Docker
+5. Qemu lite
 
-1. Create a link of Clear Containers configuration file
-
-This step is needed due to issue [#206](https://github.com/clearcontainers/runtime/issues/206)
+In Fedora (replace 25 to your version):
```

Comment:
All this can be changed with this, it will add the proper repo for the fedora version you are running. 

```
source /etc/os-release
$ sudo -E dnf config-manager --add-repo \
http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_$VERSION_ID/home:clearlinux:preview:clear-containers-2.1.repo
$ sudo dnf install qemu-lite clear-containers-image linux-container 
```